### PR TITLE
[Merged by Bors] - feat(ci): abort old jobs

### PR DESCRIPTION
**Pull request by hussein-aitlahcen**

### CODE REVIEW

File: .github/workflows/build-test-deploy.yml [hunk 1]
```diff
@@ -291,6 +316,9 @@ jobs:
       - linux
       - x64
       - sre
+    concurrency:
+      group: devcontainer-${{ github.ref }}
```
Comment (dzmitry-lahoda) [inline .github/workflows/build-test-deploy.yml:320]: ${{ github.workflow }}-${{ github.job }}-${{ github.ref }}

File: .github/workflows/check.yml [hunk 2]
```diff
@@ -28,6 +28,9 @@ env:
 jobs:
   spellcheck:
     runs-on: ubuntu-latest
+    concurrency:
+      group: testcov-${{ github.ref }}
```
Comment (dzmitry-lahoda) [inline .github/workflows/check.yml:32]:
```
${{ github.workflow }}-${{ github.job }}-${{ github.ref }}
```

**Review by dzmitry-lahoda — APPROVED** [review]

File: .github/workflows/check.yml (hunk 2 above)
Comment (hussein-aitlahcen) [reply .github/workflows/check.yml:32]: very nice

**Review by dzmitry-lahoda — APPROVED** [review]
https://stackoverflow.com/questions/61611978/get-current-job-name-in-github-actions-workflow

**Review by cor — APPROVED** [review]
LGTM

**Comment by dzmitry-lahoda** [comment]
https://stackoverflow.com/questions/71240338/obtain-job-id-from-a-workflow-run-using-contexts

**Review by dzmitry-lahoda — APPROVED** [review]

**Review by andor0 — APPROVED** [review]

**Pull request closed without merging**